dj_secure_settings/loader.py

```diff
@@ -82,7 +82,7 @@ def _load_params_from_yaml(config, yaml_params, env, namespace):
             config[k] = yaml_params[env][namespace][k]
     except KeyError:
         # couldn't load the parameters
-        logging.warning("Was not able to load params from a local YAML file.")
+        logging.debug("Was not able to load params from a local YAML file.")
```

Comment:
Similar to the comment for line 68,  could we update the message to include `env` and `namespace`? That might speed up troubleshooting when we have erroneous assumptions about those values. Also, maybe this message can say "Unable to load keys for env `env` in namespace `namespace`" or something similar, to distinguish it better from the message on line 68 and help the developer fix the issue more quickly.